eval.go

```diff
@@ -431,8 +462,11 @@ func (d *indexData) List(ctx context.Context, q query.Q) (rl *RepoList, err erro
 // include is true. Otherwise it returns an empty list.
 func (d *indexData) maybeRepoList(include bool) *RepoList {
 	l := &RepoList{}
+	l.Repos = make([]*RepoListEntry, 0, len(d.repoListEntry))
 	if include {
-		l.Repos = append(l.Repos, &d.repoListEntry)
+		for _, rle := range d.repoListEntry {
+			l.Repos = append(l.Repos, &rle)
+		}
```

Comment:
can use builtin copy

Reply:
No we cannot. `l.Repos` is a slice of values, while `d.repoListEntry` is a slice of pointers.